2019-04-11 11:27:16,996 INFO main - command line arguments: [-jvm4testexecution  /usr/lib/jvm/java-1.7.0-openjdk-amd64/bin  -mode  jgenprog  -srcjavafolder  /src/java/  -srctestfolder  /src/test/  -binjavafolder  /target/classes/  -bintestfolder  /target/test-classes/  -location  defect4j_tests/math_81  -dependencies  ./examples/libs/junit-4.4.jar  -flthreshold  0.1  -seed  10  -maxtime  100  -stopfirst  true]
Exception in thread "main" java.lang.RuntimeException: java.io.FileNotFoundException: Source 'defect4j_tests/math_81/target/classes' does not exist
	at fr.inria.astor.core.setup.ProjectRepairFacade.setupWorkingDirectories(ProjectRepairFacade.java:59)
	at fr.inria.main.evolution.AstorMain.initProject(AstorMain.java:69)
	at fr.inria.main.evolution.AstorMain.run(AstorMain.java:489)
	at fr.inria.main.evolution.AstorMain.execute(AstorMain.java:571)
	at fr.inria.main.evolution.AstorMain.main(AstorMain.java:550)
Caused by: java.io.FileNotFoundException: Source 'defect4j_tests/math_81/target/classes' does not exist
	at org.apache.commons.io.FileUtils.checkFileRequirements(FileUtils.java:1405)
	at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1368)
	at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1268)
	at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1237)
	at fr.inria.astor.core.setup.ProjectRepairFacade.copyOriginalBin(ProjectRepairFacade.java:122)
	at fr.inria.astor.core.setup.ProjectRepairFacade.setupWorkingDirectories(ProjectRepairFacade.java:54)
	... 4 more
